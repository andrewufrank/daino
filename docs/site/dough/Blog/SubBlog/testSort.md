---
title: testSort title z
author: AUF
version: publish
visibility: public
date: Jan. 4, 2018
keywords: notest
abstract: testSort abstract A silly text not needing an abstract.
# pageTemplate: page3.yaml
# image: resources/DSC08138.JPG
---

testSort text another test with title z for testing sort order

<!-- ![Alt text](DSC08138.JPG) -->
<!-- the next works -->
<!-- ![Alt text](./resources/DSC08138.JPG) -->
<!-- ![Alt text](resources/DSC08138.JPG) -->

<!-- statt einer relativen `resources/DSC08138.JPG` referenz. Problem in latex.

die absolute "/home/frank/Workspace11/ssg/docs/site/dough/Blog/SubBlog/resources/DSC08138.JPG" funktioniert. 
der file ist "/home/frank/Workspace11/ssg/docs/site/baked/Blog/SubBlog" in warum die relative nicht?

  An example post sorted at last
  and an image -->
