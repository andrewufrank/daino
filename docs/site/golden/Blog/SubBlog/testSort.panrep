Panrep {panyam = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/SubBlog/testSort.md", dyLink = "Blog/SubBlog/testSort.md", dyLang = DLenglish, dyTitle = "z", dyAbstract = "A silly text not needing an abstract.", dyAuthor = "AUF", dyDate = Just "Jan. 4, 2022", dyKeywords = "notest", dyBibliography = Nothing, dyStyle = Nothing, dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = False, dyIndexSort = Nothing, dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/SubBlog/testSort.md", link = "Blog/SubBlog/testSort.md", title = "z", abstract = "A silly text not needing an abstract.", author = "AUF", date = "Jan. 4, 2022", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []}}, panpan = Pandoc (Meta {unMeta = fromList [("abstract",MetaInlines [Str "A",Space,Str "silly",Space,Str "text",Space,Str "not",Space,Str "needing",Space,Str "an",Space,Str "abstract."]),("author",MetaInlines [Str "AUF"]),("date",MetaInlines [Str "Jan.",Space,Str "4,",Space,Str "2022"]),("keywords",MetaInlines [Str "notest"]),("pageTemplate",MetaInlines [Str "page3.yaml"]),("title",MetaInlines [Str "z"])]}) [Para [Str "another",Space,Str "test",Space,Str "with",Space,Str "title",Space,Str "z",Space,Str "for",Space,Str "testing",Space,Str "sort",Space,Str "order"],RawBlock (Format "html") "<!-- ![Alt text](DSC08138.JPG) -->",Para [Image ("",[],[]) [Str "Alt",Space,Str "text"] ("DSC08138.JPG",""),SoftBreak,RawInline (Format "html") "<!-- ![Alt text](resources/DSC08138.JPG) -->"],Para [Str "statt",Space,Str "einer",Space,Str "relativen",Space,Code ("",[],[]) "resources/DSC08138.JPG",Space,Str "referenz.",Space,Str "Problem",Space,Str "in",Space,Str "latex."],Para [Str "die",Space,Str "absolute",Space,Str "\"/home/frank/Workspace11/ssg/docs/site/dough/Blog/SubBlog/resources/DSC08138.JPG\"",Space,Str "funktioniert.",SoftBreak,Str "der",Space,Str "file",Space,Str "ist",Space,Str "\"/home/frank/Workspace11/ssg/docs/site/baked/Blog/SubBlog\"",Space,Str "in",Space,Str "warum",Space,Str "die",Space,Str "relative",Space,Str "nicht?"],Para [Str "An",Space,Str "example",Space,Str "post",Space,Str "sorted",Space,Str "at",Space,Str "last",SoftBreak,Str "and",Space,Str "an",Space,Str "image"]]}