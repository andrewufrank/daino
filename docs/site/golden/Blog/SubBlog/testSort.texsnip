TexSnip {snipyam = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/SubBlog/testSort.md", dyLink = "Blog/SubBlog/testSort.md", dyLang = DLenglish, dyTitle = "z", dyAbstract = "A silly text not needing an abstract.", dyAuthor = "AUF", dyDate = Just "Jan. 4, 2022", dyKeywords = "notest", dyBibliography = Nothing, dyStyle = Nothing, dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = False, dyIndexSort = Nothing, dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/SubBlog/testSort.md", link = "Blog/SubBlog/testSort.md", title = "z", abstract = "A silly text not needing an abstract.", author = "AUF", date = "Jan. 4, 2022", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []}}, unTexSnip = "another test with title z for testing sort order\n\n\\includegraphics{DSC08138.JPG}\n\nstatt einer relativen \\texttt{resources/DSC08138.JPG} referenz. Problem\nin latex.\n\ndie absolute\n\"/home/frank/Workspace11/ssg/docs/site/dough/Blog/SubBlog/resources/DSC08138.JPG\"\nfunktioniert. der file ist\n\"/home/frank/Workspace11/ssg/docs/site/baked/Blog/SubBlog\" in warum die\nrelative nicht?\n\nAn example post sorted at last and an image"}