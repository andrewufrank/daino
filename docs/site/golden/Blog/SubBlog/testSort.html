<!DOCTYPE html>
<!-- the master page for the pandoc templating mechanism
	needs page-title, page-title-postfix
		author date and keywords for indexing
    contentHtml, beforeContent, afterContent to wrap htmlcontent  
	uses style.css as default, other can be loaded with key css 
	-->
<html lang="DLenglish">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />


  <style type="text/css">
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      span.underline{text-decoration: underline;}
      div.column{display: inline-block; vertical-align: top; width: 50%;}
	  </style>

	
	
          <meta name="author" content="AUF" />
              <meta name="dcterms.date" content="Jan. 4, 2022" />
              <meta name="keywords" content="notest" />
    
	<title></title>
	
        <style>
            body {
            margin: 0;
            font-family: Arial, Helvetica, sans-serif;
            }

            .hero-image {
            background-image: url('');
            background-color: white;
            height: 200px;
            background-position: center;
            background-repeat: no-repeat;
            background-size: cover;
            position: relative;
            }

            .hero-text {
            transform: translate(-50%, -50%);
            color: white;
            text-shadow: 2px 2px 8px black;
            font-weight: bold;
            vertical-align: top;
            }
        </style>
    </head>

    <body>
    <!-- AF - my template for pandoc page (from the default) page3.dtpl -->
        

    <section class="hero">
        <div class="hero-image">
            
                <h1></h1>
                <h3></h3>
        <div/>
    </section>
    <section class="header">

            <section class="menu">
		<ul>
            			</ul>
    </section>
    <section>
                <header>
        <h1 class="title">z</h1>
        
                <p class="author">AUF</p>
                </header>
         
    </section>

    <section class=menu2>
		<ul>
            					<li>
				    <a href=Blog/SubBlog/testSort.html><em>z </em></a>
				    <br>A silly text not needing an abstract.		     
				    <span class=tiny>AUF "Jan. 4, 2022" 
            Blog/SubBlog/testSort.html Nothing
                    </span>
                    
    			</li>
            				</ul>
    </section>

        
<!-- blog text content section -->
    
    abstractA silly text not needing an abstract. 
    content-from-panrep-blocks<p>another test with title z for testing sort order</p>
<!-- ![Alt text](DSC08138.JPG) -->
<p><img src="DSC08138.JPG" alt="Alt text" /> <!-- ![Alt text](resources/DSC08138.JPG) --></p>
<p>statt einer relativen <code>resources/DSC08138.JPG</code> referenz. Problem in latex.</p>
<p>die absolute "/home/frank/Workspace11/ssg/docs/site/dough/Blog/SubBlog/resources/DSC08138.JPG" funktioniert. der file ist "/home/frank/Workspace11/ssg/docs/site/baked/Blog/SubBlog" in warum die relative nicht?</p>
<p>An example post sorted at last and an image</p>
    


        <p class=tiny>
        Produced with SGG  on  from  with master4.dtpl.
        </p>

            </body>


</html>
